lua/nvim-navic/init.lua

```diff
@@ -214,6 +214,7 @@ function M.get_location(opts, bufnr)
 
 	if local_config.click then
 		_G.navic_click_handler = function(minwid, cnt, _, _)
+			vim.cmd.normal("m'")
```

Comment:
Is this vim.cmd.normal added in newer neovim version? Best to use `vim.cmd("normal! m'")` to make sure we are backwords compatible